css/modals/desktop-picker/_desktop-picker.scss

```diff
@@ -26,6 +26,13 @@
             width: 30%;
         }
     }
+
+    &-spinner {
+        top: 50%;
+        margin: -15px;
```

Comment:
-15px to center it? If so I prefer translate the element by 50% left (translate(-50%, 0%)) to get the same effect without a specific pixel value.

Reply:
Thanks for the tips! I'm so bad with css :) I'll definitely give it a try!

Reply:
I didn't manage to do it with `translate(-50%, 0%)`, but I change it to not use specific pixel value! Let me know if the new approach is OK!